File: calculator/calc.py
tokens = input("what would you like to calculate?: ")
print(tokens)
x = tokens.split(',')
n = len(tokens)
print(x)
if (x[0] == '+'):
    print(sum(float(x[1:n])))
elif (x[0] == '-'):
    print(float(x[1:n]))
elif (x[0] == '*'):
    print(x[1:n])
elif (x[0] == '/'):
    print(x[1:n])
elif (x[0] == '^/'):
    print(float(x[1:n])**.5)
else:
    print('itsbroken')

File: calculator/calc.js
const readline = require('readline');

const reader = readline.createInterface({
  input: process.stdin,
  output: process.stdout,
});



reader.question("What would you like to calculate?", function(input){
	let tokens = input.split(`,`);
	//console.log (tokens)
	let mathSymbol = tokens[0];
	let num1 = Number(tokens[1]);
	let num2 = Number(tokens[2]);

	if (mathSymbol === `+`) {
		console.log (num1+num2);

	}else if (mathSymbol === `-`){
		console.log(num1-num2);
	
	}else if (mathSymbol === `*`){
		console.log(num1*num2);
	
	}else if (mathSymbol === `/`){
		console.log(num1/num2);

	}else if (mathSymbol === `^/`){
		console.log(num1**.5);
	
	}else{
		console.log (`you broke it, try entering input starting with operator then numerals seperated with commas`);
	}


	//console.log (num1, num2, mathSymbol)
	// This line closes the connection to the command line interface.
	reader.close()

});
